stdlib/public/Platform/Platform.swift

```diff
@@ -98,18 +98,29 @@ public var errno : Int32 {
   get {
```

Comment:
This errno overlay is a mess.  As a separate PR, would you mind changing our implementation strategy to be more portable?  We could have a C function that reads errno, and another one that writes to it.  These functions will use the normal `errno` macro, so they will be portable.